### PR TITLE
fix: Support ember-engines

### PR DESCRIPTION
**Pull request by jasonmit**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -23,7 +23,22 @@ module.exports = {
     });
   },
 
-  included: function(app) {
+  included: function() {
+    let app;
+
+    // If the addon has the _findHost() method (in ember-cli >= 2.7.0), we'll just
+    // use that.
+    if (typeof this._findHost === 'function') {
```
Comment (jasonmit) [inline index.js:31]: Lifted from ember-cli-mirage, handles backwards compatibility with older versions of ember-cli.

**Comment by jasonmit** [comment]
CI failures are unrelated to this change.

File: index.js (hunk 1 above)
Comment (taras) [reply index.js:31]: Is this add-on still useful?

Comment (Robdel12) [reply index.js:31]: Yeah this addon supports inline let, and ember hasn't ship that yet. https://github.com/thefrontside/ember-let/issues/49#issuecomment-430285547

I thought the same thing

Comment (taras) [reply index.js:31]: 🤔

**Pull request merged**